src/main/java/org/opensearch/neuralsearch/query/HybridQuery.java

```diff
@@ -31,20 +32,25 @@
  * Implementation of Query interface for type "hybrid". It allows execution of multiple sub-queries and collect individual
  * scores for each sub-query.
  */
+@Getter
```

Comment:
this Getter also applied to the `subQueries`, why do we need this in scope of pagination?

Reply:
We already have getter for subqueries https://github.com/opensearch-project/neural-search/blob/f844a78c045116457dc75042dca1a6af410f2df3/src/main/java/org/opensearch/neuralsearch/query/HybridQuery.java#L189.